tcs-service/pom.xml

```diff
@@ -363,6 +372,8 @@
     <tis.shared.modules.version>2.3.0</tis.shared.modules.version>
     <undertow.version>2.0.15.Final</undertow.version>
     <validation-api.version>1.1.0.Final</validation-api.version>
+
+    <sentryVersion>5.3.0</sentryVersion>
```

Comment:
I haven't checked compatibility. Is there any reason the latest version isn't being used?